pom.xml

```diff
@@ -1746,7 +1746,7 @@
     <hamcrest.version>1.3</hamcrest.version>
     <opentelemetry.version>1.0.1</opentelemetry.version>
     <opentelemetry-javaagent.version>1.0.1</opentelemetry-javaagent.version>
-    <log4j2.version>2.14.1</log4j2.version>
+    <log4j2.version>2.15.0</log4j2.version>
```

Comment:
We should handle this issue in other hbase projects also, for example hbase-operator-tools/hbase-hbck2

Reply:
Fixed the problem in hbase-operator-tools/hbase-hbck2 and hbase-operator-tools/hbase-tools.
Also checked hbase-connectors and hbase-thirdparty, did not found use of log4j2. Not sure if I missed any other related projects.

Reply:
It is fine now, thanks.